### PR TITLE
Week-7-c

### PR DESCRIPTION
**Pull request by kevie13**
Week 7 c Notes uploaded (Intro to deterministic state estimation)

### CODE REVIEW

File: bite-size-notes/week-6-c-state-estimation-intro.md [hunk 1]
```diff
@@ -0,0 +1,193 @@
+# Deterministic State Estimation
```
Comment (ShawnRrg) [inline bite-size-notes/week-6-c-state-estimation-intro.md:1]: Overall, these notes exactly covers the theory behind the state estimation, the Luenberger Observer. Let me know if you have any questions. I am Guanyu Ren.

File: bite-size-notes/week-6-c-state-estimation-intro.md [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+
+- **Definition 2:**
+
+    The pair $(A, C)$ is detectable if all the unobservable modes are stable.
+
+- **Definition 3:**
+
+    A linear system is reachable if for any $x_o, x_f \in \mathbb{R}^n$, there exists a $T>0$ and $u(t): [0, T] \rightarrow \mathbb{R}$ such that the corresponding solution satisfies $x(0) = x_o$ and $x(T) = x_f \in \mathbb{R}^n$
+
+- **Definition 4:**
```
Comment (ShawnRrg) [inline bite-size-notes/week-6-c-state-estimation-intro.md:40]: I am confused what you showed from definition 4, but overall it is really clear to show all the needed definitions before the problem starts.

File: bite-size-notes/week-6-c-state-estimation-intro.md [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+    $$
+    R = \begin{pmatrix}
+    B & AB & \dots & A^{n-1}B
+    \end{pmatrix}
+    $$
+
+### Problem Definition:
+
+Given a linear, time invariant system ignoring noise, we have state and measurement output
+
```
Comment (ShawnRrg) [inline bite-size-notes/week-6-c-state-estimation-intro.md:53]: Could you make a line changing or a space between Bu(t) and y(t)?

File: bite-size-notes/week-6-c-state-estimation-intro.md [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+$$
+\dot{\tilde{x}}(t) = \dot{x}(t) - \dot{\hat{x}}(t) = Ax(t) + Bu(t) - A\hat{x}(t) - Bu(t) = A\hat{e}(t)
+$$
+
+Now we can clearly see two situations arise from the following. Either the matrix A has all eigenvalues in the left half-plane which will allow $\hat{e}(t)$ to converge to zero. While this may converge to 0, it may be impractically slow and doesn't utilize the measurement as a feedback system. Additionally, we cannot guarantee the matrix A to be stable. Adding feedback only requires adding matrix $L$ to the estimator.
+
+$$ 
+\dot{\hat{x}}(t) = Ax(t) + Bu(t) + L(y(t) - \hat{y}(t))
+$$
+
```
Comment (ShawnRrg) [inline bite-size-notes/week-6-c-state-estimation-intro.md:89]: Could you make your expanding of our error dynamics with new estimated state dynamics more clearly? Maybe something incorrect with your editing your formula

File: bite-size-notes/week-6-c-state-estimation-intro.md [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+        &= \begin{pmatrix}
+            C^T & A^T C^T & \dots & (A^T)^{n-1} C^T
+            \end{pmatrix} \\
+        &= \begin{pmatrix}
+            C \\ CA \\ \vdots \\ CA^{n-1}
+            \end{pmatrix}^T \\
+        &= O^T
+\end{align*}
+$$
+
```
Comment (ShawnRrg) [inline bite-size-notes/week-6-c-state-estimation-intro.md:142]: For pole placement, something might be unclear with your github code.

File: bite-size-notes/week-6-c-state-estimation-intro.md [hunk 6]
```diff
@@ -0,0 +1,193 @@
...
+\end{bmatrix}
+\begin{bmatrix}
+    \dot{x}(t) \\ \tilde{x}
+\end{bmatrix}
+\end{align*}
+$$
+
+Since this matrix is upper block triangular, the eigenvalues of the combined diagonal values only depend on blaock diagonal elements. This means that the control and estimation do not interact with each other and can be design independently. As a general rule, it is advised to design the controller first by placing poles of $A-BK$ in desired LHP regions by determining $K$. Then the observer can be designed by finding $L$ to place poles of $A-LC$ in the LHP. The observer should be much faster than the controller.
+
+
```
Comment (ShawnRrg) [inline bite-size-notes/week-6-c-state-estimation-intro.md:184]: Good conclusion!

File: bite-size-notes/week-6-c-state-estimation-intro.md (hunk 1 above)
Comment (kevie13) [reply bite-size-notes/week-6-c-state-estimation-intro.md:1]: Thank you for your comments!

File: bite-size-notes/week-6-c-state-estimation-intro.md (hunk 2 above)
Comment (kevie13) [reply bite-size-notes/week-6-c-state-estimation-intro.md:40]: Fixed!

File: bite-size-notes/week-6-c-state-estimation-intro.md (hunk 3 above)
Comment (kevie13) [reply bite-size-notes/week-6-c-state-estimation-intro.md:53]: Fixed!

File: bite-size-notes/week-6-c-state-estimation-intro.md (hunk 4 above)
Comment (kevie13) [reply bite-size-notes/week-6-c-state-estimation-intro.md:89]: Fixed!

File: bite-size-notes/week-6-c-state-estimation-intro.md (hunk 5 above)
Comment (kevie13) [reply bite-size-notes/week-6-c-state-estimation-intro.md:142]: Fixed the code

File: bite-size-notes/week-6-c-state-estimation-intro.md (hunk 6 above)
Comment (kevie13) [reply bite-size-notes/week-6-c-state-estimation-intro.md:184]: Thank you!

**Pull request merged**